src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum);
+  }
+};
+
+tableHeader.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeader.children];
+  const columnName = e.target.closest('th');
+  const sortedColumn = headerChildren.indexOf(columnName);
+  const columnsToSort = [...tableBody.children];
+
+  const columnsSorted = columnsToSort.sort((a, b) => {
+    const elemA = a.children[sortedColumn].innerHTML;
+    const elemb = b.children[sortedColumn].innerHTML;
```

Comment:
There is a typo in the variable name 'elemb'; it should be 'elemB' to maintain consistency and readability.